pkgs/development/compilers/codon/default.nix

```diff
@@ -117,21 +120,22 @@ stdenv.mkDerivation {
   '';
 
   cmakeFlags = [
-    "-DCMAKE_BUILD_TYPE=Release"
-    "-DCMAKE_CXX_COMPILER=clang++"
-    "-DCMAKE_C_COMPILER=clang"
     "-DCPM_SOURCE_CACHE=${depsDir}"
     "-DLLVM_DIR=${codon-llvm}/lib/cmake/llvm"
     "-DLLVM_USE_LINKER=lld"
   ];
 
+  postInstall = lib.optionalString stdenv.isDarwin ''
+    ln -s $out/lib/codon/*.dylib $out/lib/
+  '';
+
   passthru.updateScript = nix-update-script { };
 
   meta = {
     description = "A high-performance, zero-overhead, extensible Python compiler using LLVM";
     homepage = "https://docs.exaloop.io/codon";
     maintainers = [ lib.maintainers.paveloom ];
-    license = lib.licenses.bsl11;
+    # license = lib.licenses.bsl11;
```

Comment:
What's up with this?

Reply:
Oh, right, forgot to remove after testing (the license is unfree so it's easier to comment the line).